.github/workflows/commit-message-checker.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Commit Message Checker'
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+jobs:
+  semantic-check:
+    runs-on: [self-hosted, decathlon]
+    steps:
+      - name: Check Conventional Commits specification
+        uses: gsactions/commit-message-checker@v1
+        with:
+          pattern: '^(feat|fix|refactor|test|docs|chore|style){1}(\([\w\-\.]+\))?(!)?: \b'
```

Comment:
What about multi line commit message? There isn't any official github action to check angular commit convention instead of using a custom regex? 🤔

Reply:
Yes I could use https://github.com/wagoid/commitlint-github-action instead